README.md

```diff
@@ -106,6 +106,8 @@ config.json.sample:
   "win" : {
     "title" : "Loopline Systems",
     "version" : "x.x.x.x",
+    "publisher": "Publisher Info",
+    "size": "Estimated Size",
```

Comment:
Replaced Estimated Size with a number. The units are KB as stated later on in the file.